app/serializers/initial_state_serializer.rb

```diff
@@ -22,6 +22,7 @@ def meta
       mascot: instance_presenter.mascot&.file&.url,
       profile_directory: Setting.profile_directory,
       trends: Setting.trends,
+      show_staff_badge: Setting.show_staff_badge,
```

Comment:
This shouldn't be passed down to the UI but used to decide whether or not to include the badges in the REST API response.